test/isDefined.spec.ts

```diff
@@ -0,0 +1,16 @@
+import isDefined from "../src/isDefined";
```

Comment:
```suggestion
import {isDefined} from "../src";
```

Reply:
To make sure we are doing well exporting to src
we are writing the code in the form of `../src`